hera/ui/PlayerCard.tsx

```diff
@@ -289,6 +314,61 @@ export default memo(function PlayerCard({
                   );
                 })}
             </Stack>
+            {wide && (
+              <Stack className={offsetStyle} gap nowrap start>
+                {[
+                  [
+                    'fundsPerTurn',
+                    shouldShow ? calculateFunds(map, player) : '???',
+                    <Icon
+                      className={playerStatsAfterIconStyle}
+                      icon={Reload}
+                      key={0}
+                    />,
+                  ] as const,
+                  [
+                    'netUnits',
+                    shouldShow
+                      ? Math.max(
+                          player.stats.createdUnits - player.stats.lostUnits,
+                          0,
+                        )
```

Comment:
Sorry, maybe I wasn't explaining it well earlier. We shouldn't use stats at all, they are just for internal tracking.

For units and buildings we should use the `map` object and filter `map.units` and `map.buildings` for `map.matchesPlayer(unit, player)`.

Reply:
Sorry my bad! I should've looked more into `MapData` before jumping into reading `Player` 😅

Another thing I've realized is that the initial stats for the map editor (when play-testing) now correctly reflect the number of units and buildings placed on the board!